recipes/shannon_cpp/meta.yaml

```diff
@@ -0,0 +1,44 @@
+{% set version = "0.4.0" %}
+{% set base = "https://github.com/bx3/shannon_cpp/releases/download" %}
+{% set sha256linux = "b9e70b6e64a7de64c91b83d4dfa781b149f5c121fb05cb8692fd53e22a448e0d" %}
+
+package:
+  name: shannon_cpp
+  version: '{{version}}'
+
+source:
+  - url: '{{base}}/v{{version}}/shannon_cpp-Linux64-v{{version}}.tar.gz'   # [linux]
```

Comment:
Can we simply compile this rather than using a precompiled binary?

Reply:
I can work on that, plus I am trying to have one with macOS.

Reply:
The code now compiles and should work on MacOs.